### PR TITLE
Mixed fixes - from mh's and blake's feedbacks

### PR DESCRIPTION
**Pull request by vittoriaThinkst**
# Proposed changes

## Home Page
Fix tooltip for Cards
Fix border radius
Remove focus after clicking on modal
Searchbar: reposition icon
Fix page paddings
Remove icons from navbar
Remove 'document' from token's labels

## Modal Add Token
Fix Notification inputs by adding custom arrows
Remove 'Notification settings' title
Fix border radius
Add 'required field' hint
Fix required fields for all form that needed it

Screenshot 2024-07-03 at 17 07 46
Screenshot 2024-07-03 at 17 07 17
Screenshot 2024-07-03 at 17 09 13

## Minor tweaks
Increment border radius of most elements
Fix some alignments
Fix some wording/typos

## Components
Add a new BaseLabel component + tests
Add a new BaseLabelArrow components + tests
Fix tests for BaseInput

### CODE REVIEW

File: frontend_vue/src/components/base/BaseLabelArrow.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+</template>
+
+<script setup lang="ts">
+import { ref, computed } from 'vue';
+
+const props = defineProps<{
+  id: string;
+  label: string;
+  arrowVariant?: 'one' | 'two';
+  arrowWordPosition?: number;
```
Comment (mclmax) [inline frontend_vue/src/components/base/BaseLabelArrow.vue:19]: heh, this component is very cute

to over-engineer this even further you could even see this being an array (not really feedback)

File: frontend_vue/src/components/ui/CardToken.vue [hunk 2]
```diff
@@ -25,7 +26,7 @@
       </div>
       <!--- CTA text --->
       <div
-        class="w-full leading-5 font-semibold border-t-2 border-grey-50 text-grey-700 h-[3rem] rounded-b-xl transition duration-100 hover-card shadow-solid-shadow-grey card-button justify-center items-center flex px-8"
+        class="w-full leading-5 font-semibold border-t-2 border-grey-50 text-grey-700 h-[3rem] rounded-b-2xl transition duration-100 hover-card shadow-solid-shadow-grey card-button justify-center items-center flex px-8"
       >
         {{ isHoverCard ? 'Create Token' : title }}
```
Comment (mclmax) [inline frontend_vue/src/components/ui/CardToken.vue:31]: i guess this is just github parsing this file weirdly?

**Review by mclmax — APPROVED** [review]

File: frontend_vue/src/components/base/BaseLabelArrow.vue (hunk 1 above)
Comment (vittoriaThinkst) [reply frontend_vue/src/components/base/BaseLabelArrow.vue:19]: don't tempt me...!

File: frontend_vue/src/components/ui/CardToken.vue (hunk 2 above)
Comment (vittoriaThinkst) [reply frontend_vue/src/components/ui/CardToken.vue:31]: uhm what's weird about it?

**Pull request merged**